cpp/ql/src/semmle/code/cpp/models/interfaces/FlowSource.qll

```diff
@@ -29,3 +29,12 @@ abstract class LocalFlowFunction extends Function {
    */
   abstract predicate hasLocalFlowSource(FunctionOutput output, string description);
 }
+
+/** A library function that sends data over a network connection. */
+abstract class RemoteFlowFunctionSink extends Function {
```

Comment:
The naming is asymmetric: we'd have `RemoteFlowFunction` for network sources and `RemoteFlowFunctionSink` for network sinks.

The name `RemoteFlowSource` is established across languages, so we should stick with that. It's natural, then, to have a `RemoteFlowSink`. The `Function` versions of these two should, I think, be named `RemoteFlowSourceFunction` and `RemoteFlowSinkFunction`.

Reply:
I didn't like the asymmetric naming either. I wasn't sure if we could rename `RemoteFlowFunction` so I chose to not suggest that, but if you're okay with it I fully support it! 🎉 Do we need to provide an alias for `RemoteFlowFunction` that maps to `RemoteFlowSourceFunction`?

Reply:
Thanks for making this change. But yes, we do need `RemoteFlowFunction` to remain as a `deprecated` alias to `RemoteFlowSourceFunction`.

Reply:
Fixed in 729c7f23719f494638c1fb011211cd3b67359e88.